packages/doenetml-worker-javascript/src/Core.js

```diff
@@ -2952,15 +2806,13 @@ export default class Core {
                 );
             }
         } else {
-            // since original names came from the targetComponent
-            // we can use them only if we created a new namespace
-            let originalNamesAreConsistent = newNamespace;
+            // XXX: can we always keep original names in the new regime?
+            let originalNamesAreConsistent = true;
```

Comment:
What does this flag indicate? Is there ever a time when it's false?

Reply:
It played an important role in the namespace regime. I'm pretty sure that it will go away in the new regime; hence the comment. I'll be more confident after I get through the more complicated composites like `<map>`.

Reply:
I ended up removing the code associated with this flag, as expected.